scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
-    state_exp_settings = StateExpirationSettings(max_entry_expiration=Uint32(6312000),
-                                                 min_temp_entry_expiration=Uint32(
-                                                     16),
-                                                 min_persistent_entry_expiration=Uint32(
-                                                     86400),
+
+    max_contract_size = Uint32(65536) # 64 kb
```

Comment:
I would suggest to use a single constant for both max contract size and `contract_data_entry_size_bytes` to make sure they're consistent.